odoo/addons/base/models/ir_module.py

```diff
@@ -946,18 +946,18 @@ def search_panel_select_range(self, field_name, **kwargs):
         return super(Module, self).search_panel_select_range(field_name, **kwargs)
 
     @api.model
-    def _load_module_terms(self, modules, langs, overwrite=False):
+    def _load_module_terms(self, modules, langs, overwrite=False, env=None, imported_module=False):
```

Comment:
`env` is not necessary for this api

Reply:
Indeed, thanks! I wonder if we couldn't pass self.env anytime, but I prefer this safe option.